tests/x11/disable_screensaver.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Ensure screensaver is working and then disable it
+# Maintainer: Dominik Heidler <dheidler@suse.de>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+use x11utils;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'x11';
+    ensure_unlocked_desktop;
+    assert_screen "generic-desktop";
+    # wait for screensaver to start
+    assert_screen [qw(screenlock screenlock-password)], 700;
```

Comment:
I would reduce the default timeout to 10 seconds (That way you test that the default can be changed), to not dramatically increase the duration of tests...

Reply:
reduced to 1m